src/aws_encryption_sdk/materials_managers/__init__.py

```diff
@@ -51,28 +52,87 @@ class EncryptionMaterialsRequest(object):
     )
 
 
-@attr.s(hash=False)
-class EncryptionMaterials(object):
+@attr.s
+class CryptographicMaterials(object):
+    """Cryptographic materials core.
+
+    .. versionadded:: 1.5.0
+
+    :param Algorithm algorithm: Algorithm to use for encrypting message
+    :param dict encryption_context: Encryption context tied to `encrypted_data_keys`
+    :param DataKey data_encryption_key: Plaintext data key to use for encrypting message
+    :param encrypted_data_keys: List of encrypted data keys
```

Comment:
I think that encrypted_data_keys should be in `EncryptionMaterials`.  Since Decryption does not need the EDK...

Reply:
Thinking about this, I'm inclined to agree with you on this. I wanted to keep everything in one piece, just like the `on_encrypt` path, but I agree that once the decryption materials leave the Keyring layer, the EDKs are not needed.

Reply:
No, this was addressed in a later commit on this PR....but it looks like I forgot to update the docs... :/  I'll do that.

Reply:
fixed in https://github.com/aws/aws-encryption-sdk-python/pull/163/commits/10ded57e01e1e4d130860ae94c10f5b7db397471